BICCN-policies/dataset-metadata.yaml

```diff
@@ -0,0 +1,85 @@
+identifier: REQUIRED ## Post upload (or during dandi organize)
+name: REQUIRED
+description: REQUIRED
+contributors: # required for author and contact
+- orcid: # REQUIRED
+  roles: # REQUIRED from https://casrai.org/credit/ + maintainer, contact
+  email: Recommended # filled from orcid
+  name: Recommended  # filled from orcid
+  affiliations: optional  # filled from orcid
+sponsors:
+- identifier: RECOMMENDED
+  name: REQUIRED
+  url: RECOMMENDED
+license:
+- url # REQUIRED
+keywords: [key1, key2,]
+consortium/project:
+- name: REQUIRED
+  identifier: REQUIRED #RRID
+associated_disease: # RECOMMENDED
+- name: REQUIRED
+  identifier: REQUIRED
+associated_anatomy: # RECOMMENDED
+- name: REQUIRED
+  identifier: REQUIRED
+protocols: # OPTIONAL
+- name: REQUIRED
+  identifier: REQUIRED
```

Comment:
Right now a protocol may have an identifier (specifically, a doi) - or may just have a URL.  Should we accommodate both?

Reply:
i have somewhat been using identifier synonymously for accession id's and or urls. we should clarify that.